docs/api/field_type_reference.md

```diff
@@ -1881,15 +1881,19 @@ $validators = [
 
 ## RichText Field Type
 
+!!! caution "Deprecated"
+
+    The RichText Field Type is deprecated since v2.4. Relying on any class from the `eZ\Publish\Core\FieldType` namespace is deprecated.
```

Comment:
This is half-correct, I think entire section needs to be rephrased, starting from the information that RichText Field Type is provided via eZ Platform RichTextBundle and so it should be enabled.
Then we should mention that prior version 2.4 it was provided by Kernel via `eZ\Publish\Core\FieldType\RichText` namespace which is now deprecated.